web_poet/page_inputs/client.py

```diff
@@ -191,16 +184,16 @@ async def batch_execute(
         responses despite any possible failures. This can be done by setting
         ``True`` to the ``return_exceptions`` parameter.
 
-        Like :meth:`~.HttpClient.execute`, :class:`web_poet.exceptions.http.HttpResponseError`
+        Like :meth:`~.HttpClient.execute`, :class:`~.HttpResponseError`
         will be raised for responses with status codes in the ``400-5xx`` range.
```

Comment:
I noticed that "\`\`400-5xx\`\`" and "\`\`100-3xx\`\`" are mentioned simply as "400-5xx" and "100-3xx" in some parts of the code. Let's update this to be consistent with a particular format. I'm leaning towards formatting it with "\`\`" to make them more prominent in the docs. We could also make it **bold**.

Do you have some preference on this?

Reply:
Changed them to the existing style in https://github.com/scrapinghub/web-poet/pull/50/commits/b5e869049cf15327cc0ecc9bf0ed5188b5101d9f, thanks @BurnzZ!